enhancements/declarative-index-config.md

```diff
@@ -587,7 +591,9 @@ community-operators
     │   └── ...
     └── servicemesh.json
 $ rm -r community-operators/servicemesh
-$ opm index create --from=community-operators --tag=docker.io/someothernamespace/new-community-operators:latest
+$ opm generate community-operators
```

Comment:
_technically_ you already have an index without the Dockerfile. I think the story this actually answers is more like: "I have a tool that helps me publish my index in a docker repository".

That being said, I think dropping `Dockerfile` generation altogether would make for a much better UX. An `opm build <dir|stdin>` that outputs to docker archive or a local image cache would be more straightforward and useful than scaffolding IMO.

Reply:
I thought the conclusion from various discussions were that we're not going to be shelling out to clients, and if I'm not wrong `opm build` will be doing exactly that?

Reply:
> if I'm not wrong opm build will be doing exactly that

I was imagining `opm build` __would not__ exec `docker` or `podman`, but would instead directly build archives and/or images.

Reply:
While this probably is worth discussing a little more to make sure we're confident of whichever direction we ultimately end up going, I'd prefer if we do that in a separate PR to update the EP that only touches upon this topic, so that the other updates in the EP can go in now instead of waiting for us to finish this discussion.